node/src/chain_spec.rs

```diff
@@ -152,24 +167,61 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
 /// Configure initial storage state for FRAME modules.
 fn testnet_genesis(
     wasm_binary: &[u8],
-    initial_authorities: Vec<(AuraId, GrandpaId)>,
+    initial_authorities: Vec<AuthorityKeys>,
     root_key: AccountId,
     endowed_accounts: Vec<AccountId>,
     endowed_utxos: Vec<sr25519::Public>,
     _enable_println: bool,
 ) -> GenesisConfig {
+    const ENDOWMENT: u128 = 1 << 60;
+    // minimum balance set in the runtime. check `lib.rs` of runtime module.
+    const STASH: u128 = 500;
+
     // only Alice contains 400 million coins.
-    let genesis = endowed_utxos
+    let (genesis, locked_genesis) = endowed_utxos
         .first()
         .map(|x| {
             // may need to create a const variable to represent 1_000 and 100_000_000
-            pallet_utxo::TransactionOutput::new_pubkey(
+            let alice_genesis = pallet_utxo::TransactionOutput::new_pubkey(
                 1_000 * 100_000_000 * 400_000_000 as pallet_utxo::Value,
                 H256::from_slice(x.as_slice()),
-            )
+            );
+
+            let alice_auth_keys = authority_keys_from_seed("Alice");
+
+            // since alice is the starting validator, make sure she has some utxos being locked away
```

Comment:
Can accounts be empty?

But they do need funds or UTXOs to perform a transaction, or to get a role. 
As a boot node, i think it needs money.
What would be the initial balance for each boot node?

Reply:
I guess they could. Why not split the genesis block to several UTXOs? Set aside a sk/UTXO for each boot node?

Reply:
@muursh  I never did ask; how many utxos should we create for Alice?

Reply:
We should generate new key pairs for these utxos. Let's split alice in to 5 for now each equal utxos for the genesis block. This will change in the near future but it should be trivial to do